model/src/main/resources/reference.conf

```diff
@@ -154,6 +154,24 @@ ort {
         deltaScanLimit = 10
         timeout = 60
       }
+
+      ScanOss {
+        ignoredFileExtensions = "1,2,3,4,5,6,7,8,9,ac,am,bmp,build,cfg,chm,changelog,class,cmake,conf,config"
+        ignoredFileExtensions = ${ort.scanner.options.ScanOss.ignoredFileExtensions}",contributors,copying,csproj,css"
+        ignoredFileExtensions = ${ort.scanner.options.ScanOss.ignoredFileExtensions}",csv,cvsignore,dat,data,dtd,dts"
+        ignoredFileExtensions = ${ort.scanner.options.ScanOss.ignoredFileExtensions}",dtsi,eps,geojson,gif,gitignore"
+        ignoredFileExtensions = ${ort.scanner.options.ScanOss.ignoredFileExtensions}",glif,gmo,guess,hex,html,htm,ico"
+        ignoredFileExtensions = ${ort.scanner.options.ScanOss.ignoredFileExtensions}",in,inc,info,ini,ipynb,jpg,jpeg"
+        ignoredFileExtensions = ${ort.scanner.options.ScanOss.ignoredFileExtensions}",json,license,log,m4,map,markdown"
+        ignoredFileExtensions = ${ort.scanner.options.ScanOss.ignoredFileExtensions}",md,md5,mk,makefile,meta,mxml"
+        ignoredFileExtensions = ${ort.scanner.options.ScanOss.ignoredFileExtensions}",notice,out,pdf,pem,phtml,png,po"
+        ignoredFileExtensions = ${ort.scanner.options.ScanOss.ignoredFileExtensions}",prefs,properties,readme,result"
+        ignoredFileExtensions = ${ort.scanner.options.ScanOss.ignoredFileExtensions}",rst,scss,sha,sha1,sha2,sha256,sln"
+        ignoredFileExtensions = ${ort.scanner.options.ScanOss.ignoredFileExtensions}",spec,sub,svg,svn-base,tab"
+        ignoredFileExtensions = ${ort.scanner.options.ScanOss.ignoredFileExtensions}",template,test,tex,todo,txt,utf-8"
+        ignoredFileExtensions = ${ort.scanner.options.ScanOss.ignoredFileExtensions}",version,vim,wav,xht,xhtml,xml,xpm"
+        ignoredFileExtensions = ${ort.scanner.options.ScanOss.ignoredFileExtensions}",xsd,xul,yaml,yml"
```

Comment:
Is there really no better way to configure this long list? The boilerplate is so massive that I would actually prefer one very long line with just the list in this case. Or would it maybe work if you configure it like one of the ideas below? The [HOCON documentation](https://github.com/lightbend/config/blob/main/HOCON.md#note-arrays-without-commas-or-newlines) is not very clear in that regard.

```
ignoredFileExtensions = [ 1,2,3,4,5,
    6,7,8,9,ac,
    am,...
]

ignoredFileExtensions = [ 1,2,3,4,5 ]
    [6,7,8,9,ac]
    ...
```

Reply:
> Is there really no better way to configure this long list?

Not that I'm aware of, also see [this answer](https://stackoverflow.com/a/31387467/1127485).

But I'm now hoping to be able to get rid of the blacklist completely, so I'll convert this back to a draft.